src/python/dist.py

```diff
@@ -173,16 +175,32 @@ def distribute_discrete(sizes, groups, pow=1.0):
 
     off = 0
     curweight = 0.0
-    proc = 0
+
+    all_breaks = None
+    if breaks is not None:
+        # Check that the problem makes sense
+        all_breaks = np.unique(breaks)
```

Comment:
I guess we can still call np.unique but originally it was needed because the breaks input was a list of lists.